easybuild/easyconfigs/q/QuantumESPRESSO/QE_gfortran_f90rule.patch

```diff
@@ -0,0 +1,12 @@
+diff -rupN espresso-5.4.0.orig/install/make.sys.in espresso-5.4.0/install/make.sys.in
```

Comment:
@LiangFr can you include a short description of what the patch does above this line (anything above is considered as comments), and include yourself as an author?